pages/freeboard/write/index.js

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { postArticle } from "@/api/api";
+export default function Write() {
+  const writePage = `w-full flex justify-center`;
+  const writeFrame = `w-[1200px] h-[512px] flex flex-col justify-between mt-[24px] mb-[794px]
+    md:w-[696px] md:mt-[16px] md:mb-[879px]
+    sm:w-[345px] sm:h-[410px] sm:mt-[16px] sm:mb-[965px]`;
+  const header = `w-full h-[42px] flex justify-between items-center`;
+  const titleClass = `w-[92px] h-[32px] text-[20px] leading-32px font-bold text-1f2937
+    md:leading-23.87px sm:leading-23.87px`;
+  const postBtn = `w-[74px] h-full px-[23px] py-[12px] text-[16px] font-semibold leading-19.09px text-f3f4f6 rounded-[8px]`;
+  const inputBox = `w-full h-[438px] flex flex-col
+    sm:h-[344px]`;
+  const labelClass = `h-[26px] text-[18px] leading-26px font-bold text-1f2937
+    sm:h-[24px] sm:text-[14px] leading-24px`;
+  const inputClass = `w-full h-[56px] mt-[12px] mb-[24px] px-[24px] py-[16px] rounded-[12px] focus:outline-none bg-f3f4f6
+    sm:mb-[12px]`;
+  const textAreaClass = `w-full h-[282px] mt-[12px] px-[24px] py-[16px] rounded-[12px] focus:outline-none bg-f3f4f6`;
+  const on = "bg-3692ff";
+  const off = "bg-9ca3af";
+  const [title, setTitle] = useState("");
+  const [content, setContent] = useState("");
+  const [isPost, setIsPost] = useState(false);
+  const router = useRouter();
+  const handleChangeTitle = (e) => setTitle((prev) => e.target.value);
+  const handleChangeContent = (e) => setContent(e.target.value);
+  const validate = () => {
+    if (title === "" || content === "") setIsPost(false);
+    else setIsPost(true);
+  };
+  const handleSubmitPost = async (e) => {
+    e.preventDefault();
+    const submitData = {
+      userId: "c2b44a5b-5d1f-4e6e-9b55-3f8e5e7e8b18",
+      title,
+      content
+    };
+    try {
+      const response = await postArticle(submitData);
+      router.push(`/freeboard/${response.data.id}`);
+    } catch (e) {
+      console.log(`데이터 전송 중 오류: ${e.message}`);
+    }
+  };
+  useEffect(() => {
+    validate();
+  }, [title, content]);
+  return (
+    <div className={writePage}>
+      <form className={writeFrame}>
+        <div className={header}>
+          <h1 className={titleClass}>게시글 쓰기</h1>
+          <button
+            type="button"
+            className={isPost ? `${postBtn} ${on}` : `${postBtn} ${off}`}
```

Comment:
@taeyeonkim94 
이거 아까 얘기할때 제가 태연님 말씀을 착각했던거같아요
제가 어쩔수없다고 말씀드렸던건 isEdit 가지고 삼항연산자를 만드는 부분이고, 요런 방식보다는 아래 링크에 있는 방식이였어요
https://velog.io/@shackstack/React-className%EC%97%90%EC%84%9C-%EC%82%BC%ED%95%AD%EC%97%B0%EC%82%B0%EC%9E%90-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0

```suggestion
className={`w-[74px] h-full px-[23px] py-[12px] text-[16px] font-semibold leading-19.09px text-f3f4f6 rounded-[8px] ${isPost ? 'bg-3692ff' : 'bg-9ca3a'}`}
```

Reply:
저도 이렇게 이해했습니다